packages/blocks/src/page-block/edgeless/toolbar.ts

```diff
@@ -0,0 +1,386 @@
+import { css, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import type { Page } from '@blocksuite/store';
+import {
+  ConnectorIcon,
+  DiamondIcon,
+  EllipseIcon,
+  HandIcon,
+  ImageIcon,
+  PenIcon,
+  RoundedRectangleIcon,
+  SelectIcon,
+  ShapeIcon,
+  SquareIcon,
+  TextIconLarge,
+  TriangleIcon,
+} from '@blocksuite/global/config';
+import { createPopper } from '@popperjs/core';
+import {
+  createEvent,
+  MouseMode,
+  ShapeMouseMode,
+} from '../../__internal__/index.js';
+import { assertExists } from '@blocksuite/global/utils';
+
+@customElement('shape-component')
+export class ShapeComponent extends LitElement {
```

Comment:
Consider rename to `ShapeMenu`? `ShapeComponent` may indicate shape elements or blocks.